Sources/Siwf/Models/SiwfAuthRequest.swift

```diff
@@ -25,7 +25,11 @@ public struct GenerateAuthRequest {
  */
 public enum SignedRequest {
     case siwfEncodedSignedRequest(encodedSignedRequest: String)
-    case siwfSignedRequest(requestedSignatures: SiwfRequestedSignature, requestedCredentials: [SiwfRequestedCredential] = [])
+    case siwfSignedRequest(
```

Comment:
There are some tests around this, should be fairly easy to generate a new test that covers it like this: https://github.com/ProjectLibertyLabs/siwf-sdk-ios/blob/main/Tests/SiwfTests/SiwfTests.swift#L99